src/diffusers/loaders/single_file.py

```diff
@@ -555,7 +555,4 @@ def load_module(name, value):
 
         pipe = pipeline_class(**init_kwargs)
 
-        if torch_dtype is not None:
-            pipe.to(dtype=torch_dtype)
```

Comment:
How is this handled then?

Reply:
At the model level by passing in `dtype` to their respective loading methods